editors/emacs/lambdapi-mode.el

```diff
@@ -195,6 +175,72 @@ for the goals buffer using the lambdapi-goals-window-* variables."
   (define-key lambdapi-mode-map (kbd "C-c C-b") #'lp-jump-proof-backward)
   (define-key lambdapi-mode-map (kbd "C-c C-r") #'lambdapi-refresh-window-layout))
 
+(defgroup lambdapi nil
+  "LambdaPi is a proof assistant based on the λΠ-calculus modulo rewriting"
+  :group 'languages)
+
+(defcustom lambdapi-window-X-ratio 0.5
+  "Ratio of height taken in horizontal split during window layout"
+  :type '(float)
+  :group 'lambdapi)
+
+(defcustom lambdapi-window-Y-ratio 0.8
+  "Ratio of width taken in vertical split during window layout"
+  :type '(float)
+  :group 'lambdapi)
+
+(defcustom lambdapi-window-layout '(v lambdapi-window-Y-ratio
+                                      lambdapi--temp-buffer-name
+                                      (h 0.5 "*Goals*" "*lp-logs*"))
```

Comment:
Could you please set the default layout as follows?
________
|               |
|               | text 70%
|_______ |
|               | goals 25%
|_______ |
|_______ | queries & logs 5%

Reply:
done